docs-source/content/userguide/managing-domains/model-in-image/overview.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Overview"
+date = 2020-03-11T16:45:16-05:00
+weight = 10
+pre = "<b> </b>"
+description = "Introduction to Model in Image, description of its runtime behavior, and references."
++++
+
+#### Content
+
+ - [Introduction](#introduction)
+ - [Runtime behavior overview](#runtime-behavior-overview)
+ - [Runtime updates overview](#runtime-updates-overview)
+ - [References](#references)
+
+#### Introduction
+
+Model in Image is an alternative to the operator's Domain in Image and Domain on PV domain types. See [Choose a Model]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) for a comparison of operator domain types.
```

Comment:
"domain types"?  Unfortunately, we used to call this, "models".  Note that link is choose a model.

Reply:
AI Tom: Ask Rosemary or do it yourself:   change title to 'Choosing a Domain Home Source Type'.  Change directory to 'choosing-a-domain-home-source-type'.

Reply:
Resolving. Rosemary is now working on this via a different comment.

Reply:
@tbarnes-us I did the first part: "change title to 'Choosing a Domain Home Source Type'."